src/pages/templates/page-sections/cookiePreferences/simpleCookiePreference.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Stack, Text, Button } from '@chakra-ui/react';
+import { FcLock } from 'react-icons/fc';
+
+export default function SimpleCookiePreference() {
+  return (
+    <Stack p="4" boxShadow="lg" m="4" borderRadius="sm">
+      <Stack direction="row" alignItems="center">
+        <Text fontWeight="semibold">Your Privary</Text>
+        <FcLock />
+      </Stack>
+
+      <Stack direction="row">
```

Comment:
Maybe add some responsive behaviour here? So the buttons are below the text on small viewports

Reply:
i will check this